Clean up 8/31 release notes and Datadog section (#1134)
astronomer/docs: astro/deployment-metrics.md

![DAGs page](/img/docs/dags-page.png)

> ## Export Airflow metrics to Datadog

Reviewer: Rob added this back in the previous edit, what's the impetus for removing it?

> Export over 40 Airflow metrics related to the state and performance of your Astro Deployment to [Datadog](https://www.datadoghq.com/). These metrics are in addition to the metrics that are available in the Cloud UI. They include number of queued tasks, DAG processing time, frequency of import errors, and more.

Reviewer: ```suggestion
Export over 40 Airflow metrics related to the state and performance of your Astro Deployment to [Datadog](https://www.datadoghq.com/) by adding a Datadog API key to your Deployment. For a complete list of exported metrics, see [Data Collected](https://docs.datadoghq.com/integrations/airflow/?tab=host#data-collected) in Datadog documentation.
```
"in addition to" makes it sound like the metrics have no crossover with the metrics we show in the Cloud UI is that the case?

Also instead of listing a grab bag of metrics, let's just link to the Datadog docs as soon as possible. My guess is people looking for this feature already know that they want to integrate with Datadog and just need a to know how to do it.

> After you configure this integration, the Datadog Agent automatically collects all available metrics and exports them to Datadog. For a complete list of exported metrics, see [Data Collected](https://docs.datadoghq.com/integrations/airflow/?tab=host#data-collected) in Datadog documentation.

Reviewer: I hotfixed "After you configure this integration" to make it more clear that this is just adding an API key and nothing else, I would check the current doc and update the language here accordingly.

> After you configure this integration, the Datadog Agent automatically collects all available metrics and exports them to Datadog. For a complete list of exported metrics, see [Data Collected](https://docs.datadoghq.com/integrations/airflow/?tab=host#data-collected) in Datadog documentation.

Reviewer: I think it makes more sense to have the doc link closer to the first paragraph

> After you configure this integration, the Datadog Agent automatically collects all available metrics and exports them to Datadog. For a complete list of exported metrics, see [Data Collected](https://docs.datadoghq.com/integrations/airflow/?tab=host#data-collected) in Datadog documentation.

Reviewer: ```suggestion
```

Reviewer: We can probably just cut this and talk about the Agent more below?

:::info

Astro does not export any [service checks](https://docs.datadoghq.com/integrations/airflow/?tab=host#service-checks) to Datadog. Information about the general health of your Deployment is available only as part of the Cloud UI's [Deployment health](#deployment-health) metric.

:::

1. Create a new Datadog API key or copy an existing API key. See [API and Application Keys](https://docs.datadoghq.com/account_management/api-app-keys/).
2. In the Cloud UI, select a Workspace and then select an Astro Deployment for which you want to export metrics.
3. Create a new [environment variable](environment-variable.md#set-environment-variables-in-the-cloud-ui) in your Deployment with the Datadog API key from step 1:
   - **Key:** `DATADOG_API_KEY`
   - **Value:** `<Your-Datadog-API-key>`.
   Select the **Secret?** checkbox. This ensures that your Datadog API key is saved securely and is not available to Workspace users in plain text.
4. Click **Save variable**.

> After you complete this setup, Astro launches a sidecar container in your Deployment that runs [DogStatsD](https://docs.datadoghq.com/developers/dogstatsd/?tab=hostagent). This container works with your Deployment's existing infrastructure to export Airflow metrics to the Datadog instance associated with your API key.

Reviewer 2: ```suggestion
After you complete this setup, Astro automatically launches a sidecar container in your Deployment that runs [DogStatsD](https://docs.datadoghq.com/developers/dogstatsd/?tab=hostagent). This container works with your Deployment's existing infrastructure to export Airflow metrics to the Datadog instance associated with your API key.
```

Can we emphasize that there is no other input needed from the customer in order to enable this feature? The only thing they need to do is add this variable.
